Makefile

```diff
@@ -0,0 +1,13 @@
+
+all: build-conf
+
+.PHONY: build-conf docker clean
+
+build-conf:
+	node build-nginx-configs.js
+
+docker: build-conf
+	docker-compose build
+
+clean:
+	rm out/*.conf
```

Comment:
Yeah, that's a good suggestion. It will reduce tooling and I'm not even using the dependency capabilities of `make`.